opentelemetry/CHANGELOG.md

```diff
@@ -10,6 +10,15 @@
 ### Removed
 
 - Remove `urlencoding` crate dependency. [#1613](https://github.com/open-telemetry/opentelemetry-rust/pull/1613)
+- Remove global providers for Logs [$1691](https://github.com/open-telemetry/opentelemetry-rust/pull/1691)
+    LoggerProviders are not meant for end users to get loggers from. It is only required for the log bridges.
```

Comment:
Can you add a note for log-bridge authors about how to obtain the logger and point to how the current bridges are implemened?